deps-build/darwin/aws/lib/cmake/AWSSDK/AWSSDKConfigVersion.cmake

```diff
@@ -19,13 +19,13 @@ else()
 endif()
 
 # if the installed or the using project don't have CMAKE_SIZEOF_VOID_P set, ignore it:
-if("${CMAKE_SIZEOF_VOID_P}" STREQUAL "" OR "8" STREQUAL "")
+if("${CMAKE_SIZEOF_VOID_P}" STREQUAL "" OR "4" STREQUAL "")
```

Comment:
can you elaborate how this would work for MacOSX?

Reply:
This is generated by aws sdk automatically. Not sure if how this worked.